trio/_path.py

```diff
@@ -109,30 +121,25 @@ def __dir__(self):
         return super().__dir__() + self._forward
 
     def __repr__(self):
-        return 'AsyncPath({})'.format(str(self))
+        return 'trio.Path({})'.format(str(self))
 
     def __fspath__(self):
         try:
             return self._wrapped.__fspath__()
-        except AttributeError:
+        # python3.5 compat
+        except AttributeError:  # pragma: no cover
             return str(self)
 
-    @classmethod
-    def _from_wrapped(cls, wrapped):
-        self = object.__new__(cls)
-        self._wrapped = wrapped
-        return self
-
     async def open(self, *args, **kwargs):
         func = partial(self._wrapped.open, *args, **kwargs)
         value = await trio.run_in_worker_thread(func)
         return trio.wrap_file(value)
 
 
 # not documented upstream
-delattr(AsyncPath.absolute, '__doc__')
+delattr(Path.absolute, '__doc__')
```

Comment:
Can you expand the comment here? Also, this should probably be written `del Path.absolute.__doc__`, since the attribute is a constant.